eng/scripts/automation/unified-pipeline-test.md

```diff
@@ -0,0 +1,186 @@
+# scenarios
+
+## First Onboard
+For DPG, autorestConfig will be provided in input.json as following:
+
+input.json
+
+```json
```

Comment:
Do these blocks exactly match the json test files if so perhaps you should just include a link to those files instead of duplicating the content that would need to be updated twice.

Reply:
The script only accepts json file as input, and in the input it will only can contain one json. so we cannot merge all those test json input as one and include link to it. 
Yes, it is not good to duplicate them, so I will remove unified-pipeline-test.md. thanks